keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -478,46 +487,84 @@ Because Windows privileged containers will work much differently than Linux priv
-- Mounting directories from the host OS into `hostProcess` containers will work just like with normal containers - kubelet will explicitly block this scenario.
-- All other volume types supported for normal containers on Windows will work with `hostProcess` containers.
+There will be two different behaviors for how volume mounts are configured in `hostProcess` containers.
+
+- **Bind Mounts**
+  - With the approach Window's bind-filter driver will be used to create a view that merges the host's OS filesystem with container-local volumes.
+  - When `hostProcess` containers are started a new volume will be created which contains the contents of the contaner image.
+    This volume will be mounted to `c:\hpc`.
+  - Additional volume mounts specified for `hostProcess` containers will be mounted at their requested location and can be access the same way as volume mounts in Linx or regular Windows Server contaienrs.
+    - ex: a volume with a mountPath of `/var/run/secrets/token` for containers will be mounted at `c:\var\run\secrets\token` for containers.
+  - Volume mounts will only be visible to the containers they are mounted into.
+  - The default working directory for `hostProcess` containers will also be set to `c:\hpc`.
+  - If a volume is mounted over a path that already exists on the host then the contents of the directory of the host, only the contents of the mounted volume will be visiable to the `hostProcess` container. This is the same behavior as regular Windows server behaviors.
+    - A `warn` message will be written to the containerd logs if a volume is being mounted at a location that already exists on the host.
+
+- **Symlinks**
+  - With this approach container image contents and volume mounts will be mounted at predicable paths on the host's filesystem.
+  - When `hostProcess` contaienrs are started a new volume will be created which containers the contents of the contaienr image.
+    this volume will be mounted to `c:\C\{container-id}`.
+  - Additional volumes mounts specified for `hostProcess` containers will be mounted to `c:\C\{container-id}\{mount-destination}`
+    - ex: a volume with a mountPath of `/var/run/secrets/token` for a container with id `1234` can be accessed at `c:\C\1235\var\run\secrets\token`
+  - An environment variable `$CONTAINER_SANDBOX_MOUNT_POINT` will be set to the path where the container volume is mounted (`c:\c\{container-id}`) to access content.
+    - This environment variable can be used inside the Pod manifest / command line / args for containers.
+
+A recording of the behaviors differces from a SIG-Windows community meeting can be found [here](https://youtu.be/8GeZKXgvkdY?t=309).
+Note -  In the recording it was mentioned that this functionally might not be supported on WS2019.
+    This functionality will be avaible in WS2019 but will require an OS patch (ETA: July 2022).
```

Comment:
```suggestion
    This functionality will be available in WS2019 but will require an OS patch (ETA: July 2022).
```

Reply:
will the minimum OS version be documented?

Reply:
Yes, once we know the exact patch number I'll update this doc and we'll be sure to include the minimum OS patch version for WS2019 in the K8s docs.